code/script.js

```diff
@@ -232,94 +235,175 @@ const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
-}
+  generateBoard();
+  setSecret();
+  // console.log(secret)
```

Comment:
If my memory doesn't fail me, I think Maks recommended to get rid of the console.logs and unnecessary comments once you're ready to hand in your code. It's definitely not a problem in here, but it could get cluttered quickly if you escalated your project to a bigger document :)